### PR TITLE
[NONMODULAR]Venus Humantraps are no longer capable of immortality

### PR DESCRIPTION
**Pull request by Vizoplez**
## About The Pull Request
Massively nerfs vines overall kill potential, regeneration, and ability to cheese security.

## Why It's Good For The Game
Vines are a massive issue. Sayonara hardstun beams.

## Changelog
:cl:
fix: Venus Humantraps (Should) have proper regen values
tweak: Venus Humantraps now regenerate half of what they used to per on_crossed()
tweak: Venus Humantraps have had their range reduced from 5 to 4 tiles
tweak: Venus Humantraps no longer hardstun, instead use knockdown() as intended by our combat rework
/:cl:

### CODE REVIEW

File: code/modules/mob/living/simple_animal/hostile/venus_human_trap.dm [hunk 1]
```diff
@@ -111,7 +111,7 @@
 	maxHealth = 40 //SKYRAT EDIT CHANGE
 	ranged = TRUE
 	harm_intent_damage = 5
-	obj_damage = 60
+	obj_damage = 30//Skyrat edit change - Original: 60
```
Comment (Gandalf2k15) [inline code/modules/mob/living/simple_animal/hostile/venus_human_trap.dm]: Leave this

File: code/modules/mob/living/simple_animal/hostile/venus_human_trap.dm [hunk 2]
```diff
@@ -134,7 +134,7 @@
 	/// The maximum amount of vines a plant can have at one time
 	var/max_vines = 4
 	/// How far away a plant can attach a vine to something
-	var/vine_grab_distance = 5
+	var/vine_grab_distance = 2 //SKYRAT EDIT - Original 5
```
Comment (Gandalf2k15) [inline code/modules/mob/living/simple_animal/hostile/venus_human_trap.dm]: Too small, make it 4

File: code/modules/mob/living/simple_animal/hostile/venus_human_trap.dm [hunk 3]
```diff
@@ -174,7 +174,7 @@
 	vines += newVine
 	if(isliving(the_target))
 		var/mob/living/L = the_target
-		L.Paralyze(20)
+		L.Knockdown(2 SECONDS) //Skyrat EDIT - Removes hardstun, bye!
```
Comment (Gandalf2k15) [inline code/modules/mob/living/simple_animal/hostile/venus_human_trap.dm:177]: This is fine because it makes people drop weapons.

Comment (Vizoplez) [reply code/modules/mob/living/simple_animal/hostile/venus_human_trap.dm:177]: Paralyse prevents picking up weapons, and 20 ticks on average lag ~4 seconds of a hardstun where you cannot do anything. This also worked on simplemobs, versus knockdown which does not.

File: code/modules/mob/living/simple_animal/hostile/venus_human_trap.dm (hunk 2 above)
Comment (Vizoplez) [reply code/modules/mob/living/simple_animal/hostile/venus_human_trap.dm]: I'm making it 3, so they cant retreat deep into vines and spam it again, and hopefully reduce instances of cornerbanging.

**Pull request merged**